imagesets/generic-worker-ubuntu-24-04-staging/bootstrap.sh

```
@@ -166,6 +166,9 @@ if [ '%MY_CLOUD%' == 'aws' ]; then
   echo 'snd-aloop' >> /etc/modules
 fi
 
+# avoid unnecessary shutdowns during worker startups
+systemctl disable unattended-upgrades
+
```

This is nice and simple. Let's test there are no issues reported in `/var/log/bootstrap.<whatever>` (i forget file path) in GCP and AWS to be sure this doesn't silently fail etc.